sakai-evaluation-tool/src/webapp/component-templates/hierarchy_controls.html

```diff
@@ -93,8 +94,15 @@
                     <a rsf:id="assign-groups-link" href="" style="font-size: 0.9em;">Assign Groups</a>
                 </td>
                 <td rsf:id="users-cell">
+                    <span rsf:id="assign-user-count">67</span>
+                    &nbsp;
```

Comment:
Although I do agree with you, this pattern is used heavily in EvalSys. Instead of fixing this one off occurrence, we should maybe open a new ticket to address all uses of `&nbsp;` in the project.